### ISSUE TITLE
WCAG 1.4.3: Elements must have sufficient color contrast (.form-control-feedback)

### ISSUE THREAD

**Issue by brianteeman** [joomla/backend-template#99]
Tags: Accessibility, WCAG 1.4.3, color-contrast

Issue: Elements must have sufficient color contrast (color-contrast: https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=msftAI)

Target application title: backend design - Administrator Login
Target application url: http://localhost/backend-template-release/backend-template-release/administrator/index.php

Element path: .form-control-feedback

Snippet: `<span class="form-control-feedback">Field value cannot be empty</span>`

How to fix:
Fix any of the following:
  Element has insufficient color contrast of 1.45 (foreground color: #cbd8e1, background color: #ffffff, font size: 9.0pt, font weight: normal). Expected contrast ratio of 4.5:1

**Comment by chmst** [comment]
Closed as solved with PR #131
